custom/enikshay/integrations/bets/repeaters.py

```diff
@@ -305,17 +307,25 @@ class BETSLocationRepeater(LocationRepeater):
                         'pcc',
                         'pac',
     )
+    return (location.metadata.get('is_test') != "yes"
+            and location.location_type.code in location_types_to_forward)
 
-    @classmethod
-    def available_for_domain(cls, domain):
-        return BETS_INTEGRATION.enabled(domain)
 
-    def allowed_to_forward(self, location):
-        return (location.metadata.get('is_test') != "yes"
-                and location.location_type.code in self.location_types_to_forward)
+class BETSUserRepeater(BETSRepeaterMixin, UserRepeater):
+    friendly_name = _("Forward users to BETS")
+    payload_generator_classes = (UserPayloadGenerator,)
 
-    class Meta(object):
-        app_label = 'repeaters'
+    def allowed_to_forward(self, user):
+        return any(_is_real_private_location(loc)
+                   for loc in user.get_sql_locations(self.domain))
```

Comment:
@proteusvacuum What's the appropriate check here?  User assigned to a location with type in `['tu', 'plc', 'pcp', 'pcc', 'pac',]`?